config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -0,0 +1,19 @@
+presubmits:
+  kubernetes/cloud-provider-gcp:
+  - name: cloud-provider-gcp-tests
+    agent: kubernetes
+    max_concurrency: 5
+    always_run: true
+    context: cloud-provider-gcp-tests
+    rerun_command: "/test cloud-provider-gcp-tests"
+    trigger: "(?m)^/test( all| cloud-provider-gcp-tests),?(\\s+|$)"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/cloud-builders/bazel
+        entrypoint: bash
```

Comment:
Can you change this to command? By setting `decorate: true` prow currently rewrites your entrypoint

https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#how-to-configure